requirements.txt

```diff
@@ -21,4 +21,5 @@ Flask
 Flask-CacheBuster
 flask-cors
 waitress
-lxml
\ No newline at end of file
+lxml
+packaging
```

Comment:
hey @robintw - `packaging` was a good find :-)

Reply:
I was really confused why it wasn't in our requirements.txt already, as I swear I remember adding it - then I remembered that I'd used it in one of my other open-source projects recently, and had got confused!